Analyse_Birds.qmd

```diff
@@ -12,57 +12,419 @@ output:
+- *Habitat* = Grassy habitat in which nest was located (pasture, cool-season grassland, warm-season grassland, or prairie)
+- *Species* = English common name
+- *Year* = Year nest was found
+- *Nest_Fate* = Successly fledged at least one young (1) or Failed (0).
+- *Initiation_Date* = Coded date when the first egg was laid. April 1 = 1; May 1 = 31; June 1 = 62; July 1 = 92; August 1 = 123; September 1 = 154
+- *Number_Fledged* = For a successful nest, number of young in nest on nest end date; For a failed nest, Number_Fledged is 0
+- *Nest_End_Date* = Coded date when the nest failed or was successful. April 1 = 1; May 1 = 31; June 1 = 62; July 1 = 92; August 1 = 123; September 1 = 154
 
-# Document R modifié et avec ajouts
+# What kind of questions can we rise ?
 
-## Import des données
 
-**Do not forget to set your working directory in the right place**
-**Menu : Session/Set Working Directory/Choose directory**
-*And indicate Git/2024_MODE_OCR*
+```{r,include=FALSE}
+# Datatset import under the name of "data"
+data <- read.csv("Bird_dataset.csv", sep=";", header = TRUE)
 
+# Selecting the data of interest (here our study is going to focus on the state of illinois)
+data_Illinois <- subset(data, data$State == "Illinois")
 
-```{r}
-# Datatset import under the name of "data"
-data <- read.table("Bird_dataset.csv", sep="\t", header = TRUE)
+# Converting explanatory variables (or covariates) into factor (they were intially character chain )
+data_Illinois$County <- as.factor(data_Illinois$County)
+data_Illinois$Habitat <- as.factor(data_Illinois$Habitat)
+data_Illinois$Species <- as.factor(data_Illinois$Species)
+data_Illinois$Year <- as.factor(data_Illinois$Year)
+
+# checking for missing values
+colSums(is.na(data_Illinois))
 ```
 
-```{r}
-# Charge the librairies
-library(ggplot2)
+
+
+```{r, include=FALSE}
+library(knitr)
+opts_chunk$set(echo = FALSE, comment = "", cache = TRUE, fig.align = "center")
+library(ggplot2) # graph package
+library(tinytex) # Pour la sortie pdf
+library(corrplot)# Correlation matrix calculus
+library(plot3D)# For 3D plot
+library(DHARMa)# Model diagnosis
+library(rcompanion)# Model pseudo R²
+library(lattice)# Multipanel graphics
+```
+
+
+
+
+# Analysing the relationship between nest fate and county, habitat, species, year, initiation date using a generalized linear model (GLM) with a binomial distribution
+
+## I. Data Exploration
+
+Variables are :  
+
+- *Nest_Fate* : Response variable, binary
+- *County* : qualitative explanatory variable
+- *Habitat* : qualitative explanatory variable
+- *Species* : qualitative explanatory variable
+- *Year* : qualitative explanatory variable
+- *Initiation_Date* : Quantitative explanatory variable
+
+**Aim of the study** : We want to model the faye of the nest as a response of the County, the Habitat, the Species, the Year, the Initiation date. We are looking for a model that could explain the distribution of Nest fate. 
```

Comment:
Mistake : you wrote "faye" instead of "fate" I think

Reply:
corrected